### ISSUE TITLE
This tool is great!

### ISSUE THREAD

**Issue by thecrazy** [willnode/MeshDebugger#1]
Just love it!

I finally have a tool that I can use to show modelers that the problem is on their side! ( just kidding ;P )

I may add a few things, Ill let you know if I do!

Thank you so much!

**Comment by bradzoob** [comment]
Yeah, this is an incredible tool, thank you so much for making this :)

**Comment by bradzoob** [comment]
Also something worth mentioning, i ran this in the latest 2020 release without issue.

**Comment by RedDude** [comment]
Amazing tool! I'm working with Halfedge meshes and this save me tons of hours. willnode is such a hero
